lib/manageiq/environment.rb

```diff
@@ -61,7 +61,8 @@ def self.setup_gemfile_lock
       # Gemfile.lock.release only applies to non-master branches and PRs to non-master branches
       return unless ENV["GITHUB_REPOSITORY_OWNER"] == "ManageIQ" &&
                     ENV["GITHUB_BASE_REF"] != "master" && # PR to non-master branch
-                    ENV["GITHUB_REF_NAME"] != "master"    # A non-master branch
+                    ENV["GITHUB_REF_NAME"] != "master" && # A non-master branch
+                    !ENV["GITHUB_REF_NAME"].to_s.start_with?("dependabot/") # Dependabot makes branches in the core repo
```

Comment:
Seems like we could make this more generic rather than specific to dependabot if we flip the logic from (is it not master and not this and not that) to "is it a release branch" but since we try hard not to have non release branches on the main fork this will work for now

Reply:
The problem is how to define "release branch" without having to enumerate them all.

Reply:
even just having a method called `release_branch?` with these ugly contents would be more helpful here.

Reply:
Incidentally, this is one of the reasons I want to use core's before_install like in https://github.com/ManageIQ/manageiq-providers-amazon/pull/744, so I can delete this method entirely and we can just rely on the bin/before_install script's version.

Reply:
> even just having a method called release_branch? with these ugly contents would be more helpful here.

Agreed, but I'm hoping to delete it altogether.